app/src/main/java/com/github/campus_capture/bootcamp/fragments/MapsFragment.java

```diff
@@ -90,14 +108,17 @@ public class MapsFragment extends Fragment{
      * Flag indicating whether a requested permission has been denied
      */
     private boolean permissionDenied = false;
+    @SuppressLint("PotentialBehaviorOverride")
     private final OnMapReadyCallback callback = googleMap -> {
         ZoneDAO zoneDAO = zoneDB.zoneDAO();
 
         map = googleMap;
+        map.setOnCameraMoveListener(this);
         UiSettings mapUiSettings = map.getUiSettings();
 
-        enableMyLocation();
-
+        map.setMaxZoomPreference(18);
```

Comment:
The zoom conditions are quite annoying... We will maybe have to find something better for the names but for now this is OK.